### ISSUE TITLE
Feed Me fails to populate a Hyper link

### ISSUE THREAD

**Issue by cornedor** [verbb/hyper#153]
### Describe the bug

Feed Me fails to run when I try to populate a Hyper link that is inside a Matrix entry. It works fine when the hyper field is in the entry itself.

Maybe related (if not, I'll create a new bug report), when I add the Hyper field in the entry itself *and* in the Matrix entry, they both get assigned the same mapping.

`$this->field` is undefined in https://github.com/verbb/hyper/blob/craft-5/src/integrations/feedme/fields/Hyper.php#L38

```
TypeError: Cake\Utility\Hash::get(): Argument #1 ($data) must be of type ArrayAccess|array, null given, called in /var/www/html/vendor/verbb/hyper/src/integrations/feedme/fields/Hyper.php on line 38 and defined in /var/www/html/vendor/cakephp/utility/Hash.php:55
Stack trace:
#0 /var/www/html/vendor/verbb/hyper/src/integrations/feedme/fields/Hyper.php(38): Cake\Utility\Hash::get(NULL, 'settings.multip...')
#1 /var/www/html/vendor/craftcms/feed-me/src/services/Fields.php(234): verbb\hyper\integrations\feedme\fields\Hyper->parseField()
#2 /var/www/html/vendor/craftcms/feed-me/src/services/Process.php(405): craft\feedme\services\Fields->parseField(Array, Object(craft\elements\Entry), Array, 'linkTo', Array)
#3 /var/www/html/vendor/craftcms/feed-me/src/services/Process.php(624): craft\feedme\services\Process->processFeed(0, Array, Array)
#4 /var/www/html/vendor/craftcms/feed-me/src/controllers/FeedsController.php(296): craft\feedme\services\Process->debugFeed(Object(craft\feedme\models\FeedModel), NULL, NULL, Array)
#5 [internal function]: craft\feedme\controllers\FeedsController->actionDebug()
#6 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#7 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#8 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('debug', Array)
#9 /var/www/html/vendor/craftcms/cms/src/web/Application.php(349): yii\base\Module->runAction('feed-me/feeds/d...', Array)
#10 /var/www/html/vendor/craftcms/cms/src/web/Application.php(650): craft\web\Application->runAction('feed-me/feeds/d...', Array)
#11 /var/www/html/vendor/craftcms/cms/src/web/Application.php(311): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#12 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#13 /var/www/html/web/index.php(12): yii\base\Application->run()
#14 {main}
```

![SCR-20240508-mtrf](https://github.com/verbb/hyper/assets/570297/6644ac35-2d2c-46f4-90c7-cff5c2ae3666)

### Steps to reproduce

1. Create a Matrix entry, and add a Hyper field
2. Use Feed Me to import data into the Hyper field.

### Craft CMS version

5.1.2

### Plugin version

2.0.0-beta.8

### Multi-site?

Yes

### Additional context

Feed Me 6.0.1

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/hyper:"dev-craft-5 as 2.0.0-beta.8"`.

**Comment by engram-design** [comment]
Fixed in [2.0.0](https://github.com/verbb/hyper/releases/tag/2.0.0)
